### DISCUSSION TITLE
Problem with the Model Interface and Excel

### DISCUSSION THREAD

**Discussion by kke13** [JGCRI/gcam-core#484] [Q&A]
The problem starts when I copy the numbers of Model Interface and paste them on Excel as they do not paste properly and they are very big. I have tried to fix it by many different ways but it did not work well. Some of them are the above. Firstly, I changed the computer settings with the comma and dot thinking that may be a problem. Secondly, I changed the decimal and thousands separator of Excel from File --> Options --> Advanced Options for working with Excel, but neither this change did affect the numbers.

**Comment by ssmithClimate** [comment]
You can drag and drop the results of a query in the model interface into Excel. So click and hold on the title of the query result (for example "resource production" in the image) and drag into Excel. 
<img width="483" alt="image" src="https://github.com/user-attachments/assets/060f9fee-e1f9-4127-b054-8701071c6163">

If you're getting a lot of data from a GCAM DB, you might want to consider using the [rgcam](https://github.com/JGCRI/rgcam) package.

**Reply by kke13** [reply]
Even though I drag and drop the results, the numbers are not correct.

**Reply by DLoughlin** [reply]
Could you provide screen shots of the data in the ModelInterface and what you are seeing in Excel? That would be helpful in understanding the problem. Thank you.
